src/SessionManager/ProgramSettings.cc

```diff
@@ -137,6 +137,7 @@ void ProgramSettings::ApplyCommandLineSettings(int argc, char** argv) {
         ("log_protocol_messages", "enable protocol message debug logs", cxxopts::value<bool>())
         ("no_http", "disable frontend HTTP server", cxxopts::value<bool>())
         ("no_browser", "don't open the frontend URL in a browser on startup", cxxopts::value<bool>())
+        ("browser", "user custom browser + args", cxxopts::value<string>(), "<browser>")
```

Comment:
I thought the intention was to merge these into a single option, and use `--browser=none`, but maybe it's fine to leave this as two options. On the one hand, I would prefer not to have a proliferation of related and overlapping options. On the other hand, checking explicitly for a "none" keyword is a bit hacky.

We should add in the body of the help that `no_browser` takes precedence over `browser`.

Reply:
initially I'd wanted that too but @jolopezl convinced me otherwise :smile:

FWIW jupyter also uses two separate args

Reply:
Makes sense.